config/core/test-configs.yaml

```diff
@@ -2025,6 +2025,13 @@ test_configs:
   - device_type: imx6q-sabrelite
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
```

Comment:
Failed: https://staging.kernelci.org/test/plan/id/62337904137c97d1e14b3155/

Reply:
Timeout occurred in this case. Maybe some testcase failed. In the last couple of weeks, this test hasn't run on this target again. So we aren't sure if this is a problem or not.